### PR TITLE
Notification: Number Badge on Listview

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] https://github.com/Samsung/TAU/issues/1539
[Problem] Implement number badge on listview acording to guideline
[Solution]
     - added new css
     - new example page has added to UIComponents

[Screenshot]
obraz

obraz

    

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

File: examples/mobile/UIComponents/components/notification-badge/number-badge-appbar.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+					</option>
+				</select>
+			</div>
+		</header>
+
+		<div class="ui-content">
+		</div>
+
+		<script>
+			var dropDownMenu = document.getElementById("sample-dropdown-menu");
```
Comment (DohyungLim) [inline examples/mobile/UIComponents/components/notification-badge/number-badge-appbar.html]: Why don't you move the content to a js file?

**Review by DohyungLim — COMMENTED** [review]
1 comment.

**Review by HunseopJeong — APPROVED** [review]
LGTM

**Comment by TomaszLukawskiSam** [comment]
> LGTM

Thank you for review :)

**Pull request merged**